contracts/contracts/vault/VaultCore.sol

```diff
@@ -390,6 +395,77 @@ contract VaultCore is VaultStorage {
         return vaultValue;
     }
 
+    /**
+     * @notice Strategist swaps collateral assets sitting in the vault.
+     * @param _fromAsset The token address of the asset being sold by the vault.
+     * @param _toAsset The token address of the asset being purchased by the vault.
+     * @param _fromAssetAmount The amount of assets being sold by the vault.
+     * @param _minToAssetAmount The minimum amount of assets to be purchased.
+     * @param _data implementation specific data. eg 1Inch swap data
+     * @return toAssetAmount The amount of toAssets that was received from the swap
+     */
+    function swapCollateral(
```

Comment:
Given that this is a admin function, can it go in VaultAdmin.sol? If there's not room, we can remove reallocate.

Reply:
Good point. I'll move